angular-workspace/projects/ni/nimble-angular/table/testing/ng-package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "../../../../../../node_modules/ng-packagr/ng-package.schema.json",
```

Comment:
Just to double check, do we need to specify these schema attributes? it's a bit of a bummer that we need to link out to the workspace root but not a deal breaker.

Reply:
sounds handy enough then 👍